src/sass/layouts/_header.scss

```diff
@@ -0,0 +1,212 @@
+.header {
+  backdrop-filter: blur(1rem);
+  display: flex;
+  position: fixed;
+  z-index: 100;
+  max-height: 8rem;
+  width: 100%;
+  padding: 2rem 4rem;
+  justify-content: space-between;
+  align-items: center;
+  background-image: linear-gradient(
+    to right,
+    rgb(55 65 81 / 30%),
+    rgb(55 65 81 / 60%)
+  );
+
+  @include respond(tab-port) {
+    padding: 2rem 3rem;
+  }
+
+  @include respond(phone) {
+    padding: 2rem;
+  }
+
+  &__logo-container {
+    flex: 0 0 5%;
+    min-width: 5.5rem;
+
+    &__img {
+      height: 100%;
+      max-width: 100%;
+    }
+  }
+
+  &__btn-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+
+    @include respond(phone) {
+      flex-direction: column;
+      justify-content: center;
+      height: 100vh;
+      width: 60%;
+      position: absolute;
+      left: 100%;
+      top: 0;
+      z-index: 101;
```

Comment:
why 101 only, why not just 1?

Reply:
Just gave it a random high number to ensure it stays on top. I will make variables instead to make it more consistent.